### PR TITLE
Various fixes and improvements to the custom keybindings

### PR DESCRIPTION
**Pull request by ThatLing**
Originally did this for /tg/ but they have actual code standards, bleh.

Adds more keys, replaces the million hrefs with a single href, fixes looking around using arrow keys, fixes no keys on first connect, fixes keys being broken with default mode.

### CODE REVIEW

File: code/__DEFINES/~yogs_defines/keybindings.dm [hunk 1]
```diff
@@ -64,6 +64,18 @@ GLOBAL_LIST_INIT(keybinding_validkeys, list(
 	"Numpad9",
 	",",
 	".",
+	"&",
+	"é",
+	"\"",
+	"'",
+	"(",
+	"§",
+	"è",
```
Comment (Amelia0010) [inline code/__DEFINES/~yogs_defines/keybindings.dm:73]: 🇫🇷

File: code/__DEFINES/~yogs_defines/keybindings.dm [hunk 2]
```diff
@@ -64,6 +64,18 @@ GLOBAL_LIST_INIT(keybinding_validkeys, list(
...
+	"!",
+	"ç",
```
Comment (Amelia0010) [inline code/__DEFINES/~yogs_defines/keybindings.dm:75]: 🇵🇹

File: code/__DEFINES/~yogs_defines/keybindings.dm (hunk 1 above)
Comment (ThatLing) [reply code/__DEFINES/~yogs_defines/keybindings.dm:73]: shut up

File: code/__DEFINES/~yogs_defines/keybindings.dm (hunk 2 above)
Comment (ThatLing) [reply code/__DEFINES/~yogs_defines/keybindings.dm:75]: yes

**Pull request merged**